src/main/java/database/RoomDBModel.java

```diff
@@ -0,0 +1,42 @@
+package database;
+
+import java.util.List;
+
+public class RoomDBModel {
+    private final int roomID;
+    private final String roomName;
+    private final Integer hostID;
+    private List<Integer> activeUserIDs;
+    private String messageHistory;
+
+    public RoomDBModel(int roomID, String roomName, Integer hostID, List<Integer> activeUserIDs, String messageHistory) {
```

Comment:
Long parameter code smell. How about introducing parameter object or using factory pattern?

Reply:
can you just pass a room entity